### PR TITLE
Feat custom forms

### PR DESCRIPTION
**Pull request by rishi9684**

### CODE REVIEW

File: pages/apps/_app/event/_id/editSiteSetting.vue [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      </v-card>
+    </v-dialog>
+  </v-layout>
+</template>
+
+<script>
+export default {
+  props: {
+    // eslint-disable-next-line vue/require-prop-types
+    sitesetting: {
```
Comment (SukantGujar) [inline pages/apps/_app/event/_id/editSiteSetting.vue:161]: camelCase the prop names.

File: pages/apps/_app/event/_id/newBadgeForm.vue [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </v-layout>
+</template>
+
+<script>
+export default {
+  props: {
+    // eslint-disable-next-line vue/require-prop-types
```
Comment (SukantGujar) [inline pages/apps/_app/event/_id/newBadgeForm.vue:69]: Why is this disable flag added? Also, newbadge should be camelCase

File: pages/event/_id/index.vue [hunk 3]
```diff
@@ -453,6 +682,9 @@ export default {
   data() {
     return {
       loading: 0,
+      editeventform: false,
```
Comment (SukantGujar) [inline pages/event/_id/index.vue:685]: Casing please

**Review by SukantGujar — CHANGES_REQUESTED** [review]
Casing needs to be corrected in a lot of places. I have marked a few of them.

File: pages/apps/_app/event/_id/editSiteSetting.vue (hunk 1 above)
Comment (rishi9684) [reply pages/apps/_app/event/_id/editSiteSetting.vue]: camelcase done

File: pages/event/_id/index.vue (hunk 3 above)
Comment (rishi9684) [reply pages/event/_id/index.vue:685]: done

File: pages/apps/_app/event/_id/newBadgeForm.vue (hunk 2 above)
Comment (rishi9684) [reply pages/apps/_app/event/_id/newBadgeForm.vue]: removed flag

**Review by bitpod-io — APPROVED** [review]

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**